channel.go

```diff
@@ -278,8 +278,9 @@ func (ch *Channel) dispatch(msg message) {
 		// to avoid unexpected interleaving with basic.publish frames if
 		// publishing is happening concurrently
 		ch.m.Lock()
-		// TODO check error case
-		ch.send(&channelCloseOk{}) //nolint
+		if err := ch.send(&channelCloseOk{}); err != nil {
+			ErrLog.Printf("channel send error: %+v", err)
```

Comment:
All log message should:
-  include the context of the error, e.g. that it happens while trying to close the channel
- information that allows to relate log messages to a specific client and channel if multiple clients exist it won't be clear on which one the error happened